<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controle de Presença</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>Controle</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP da máquina</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="serverIP">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Disciplina</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="disCode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="classCode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Turma</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Data</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="date">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>110</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="connect">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Iniciar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>340</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Salvar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="create">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Criar</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="presenceList">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>10</y>
      <width>461</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNovo_aluno">
   <property name="text">
    <string>Novo aluno</string>
   </property>
  </action>
  <action name="actionNova_turma">
   <property name="text">
    <string>Nova turma</string>
   </property>
  </action>
  <action name="actionNova_turma_2">
   <property name="text">
    <string>Nova turma</string>
   </property>
  </action>
  <action name="actionNovo_aluno_2">
   <property name="text">
    <string>Novo aluno</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
